web_action_conditionable/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl
+    :alt: License: AGPL-3
+
+Web Action Conditionable
+========================
+
+This module was written to extend the functionality of actions in tree views.
+Odoo by default support:
+
+::
+
+   <tree delete="false" create="false">
+
+with this module you can:
+
+::
+   
+   <tree delete="state=='draft'">
+
+It works in any tree view, so you can use it in One2many.
+
+
+Installation
+============
+
+To install this module, just follow basic steps to install an odoo module.
+
+
+Configuration
+=============
+
+Not needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/11.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
```

Comment:
please, update to latest [README](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)